ethereum/core/src/main/java/org/hyperledger/besu/ethereum/MainnetBlockValidator.java

```diff
@@ -246,15 +251,24 @@ public boolean validateBlockForSyncing(
       final HeaderValidationMode headerValidationMode,
       final HeaderValidationMode ommerValidationMode,
       final BodyValidationMode bodyValidationMode) {
+
+    if (bodyValidationMode == BodyValidationMode.FULL) {
```

Comment:
Why is full body validation not supported?

Reply:
The only external path that calls `validateBodyLight` is `validateBlockForSyncing`, which will only be `NONE` or `LIGHT`, depending on PoS. In this path. As we do not have the `stateRoot`, I decided to enforce it so if something changes we will have to explicitly change this method.